rules/0131.go

```diff
@@ -71,6 +73,84 @@ func checkGetRequestMessageName() lint.Rule {
 	}
 }
 
+// Get messages should have an appropriate (google.api.http).get URI.
+func checkGetURI() lint.Rule {
+	return &descriptor.CallbackRule{
+		RuleInfo: lint.RuleInfo{
+			Name:         lint.NewRuleName("core", "0131", "uri"),
+			Description:  "check that Get RPCs have an appropriate URI",
+			RequestTypes: []lint.RequestType{lint.ProtoRequest},
+			URI:          "https://aip.dev/131#guidance",
+		},
+		Callback: descriptor.Callbacks{
+			MethodCallback: func(m p.MethodDescriptor, s lint.DescriptorSource) (problems []lint.Problem, err error) {
+				// We only care about Get methods for the purpose of this rule;
+				// ignore everything else.
+				if !isGetMethod(m) {
+					return
+				}
+
+				// Iterate over the HTTP rules.
+				for _, httpRule := range getHTTPRules(m) {
+					// Rule check: Ensure that the GET HTTP verb is used.
+					getURI := httpRule.GetGet()
+					if getURI == "" {
+						problems = append(problems, lint.Problem{
+							Message:    "Get RPCs should use the GET HTTP verb.",
```

Comment:
I haven't been too thorough. Are you using "should" vs "must" as appropriate in these error messages?

Reply:
Probably not. Good call.

Reply:
This one definitely should be must.

```suggestion
							Message:    "Get RPCs must use the GET HTTP verb.",
```